pkgs/tools/misc/yubikey-manager-qt/default.nix

```diff
@@ -27,30 +24,29 @@ stdenv.mkDerivation rec {
     sha256 = "087ms9i0n3rm8a0hvc4a2dk3rffbm6rmgz0m8gbjk6g37iml6nb7";
   };
 
-  nativeBuildInputs = [ wrapQtAppsHook python3.pkgs.wrapPython qmake ];
-
   postPatch = ''
-    substituteInPlace ykman-gui/deployment.pri --replace '/usr/bin' "$out/bin"
+    substituteInPlace ykman-gui/deployment.pri \
+      --replace /usr/bin $out/bin
```

Comment:
Why did you change this line? Did you ran some linter tool? But then, why didn't you also change a similar line for the `substituteInPlace` call at line 54?

Usually, I am in favor of proper quoting, to be safe. But at least it should be consistent in the same file.